src/server.js

```diff
@@ -37,7 +37,10 @@ app.get("/api/todo", async (req, res) => {
 });
 
 app.use(express.json());
-app.post("/api/todo", async (req, res, next) => {
+
+// POST is for creating a new resource
+// add endpoints for PATCH, DELETE to perform other operations
+app.post("/api/todo", async (req, res) => {
```

Comment:
as noted elsewhere this handler either needs to be split into three (recommended) or the object coming in  needs to be updated